app/controllers/agents/users_controller.rb

```diff
@@ -94,6 +90,15 @@ def link_to_organisation
 
   private
 
+  def create_from_modal
+    if @user_to_compare.present?
+      @user.errors.add(:base, "L'utilisateur que vous essayez de créer existe deja")
```

Comment:
I think it's worth it trying to move this to a proper validation, it will make more sense and remove code from the controller and maybe remove the need for this `create_from_modal` method